fpfilter.pl

```diff
@@ -542,10 +544,11 @@ sub filter_sites_in_hash {
 
 sub setup_workdir {
     my ($reference, $bam_file, $bam_index) = @_;
+
     $reference = abs_path($reference);
     $bam_file = abs_path($bam_file);
     $bam_index = abs_path($bam_index) if $bam_index;
-
+    
```

Comment:
sorry, using vim instead of emacs for this.  Must have hit the wrong key at the wrong time.